pages/pipelines/managing_log_output.md.erb

```diff
@@ -110,3 +110,14 @@ xcodebuild <build options> | tee -a build.log | xcpretty -c
 Make sure to set the `-o pipefail` option in your buildscript as above, otherwise the build failure status might not be passed through correctly.
 
 <%= image("xcpretty.png", alt: "Screenshot of xcpretty output", size: '656x316') %>
+
+## Redacted environment variables
+
+Environment variables matching common patterns for passwords and other secure information are redacted before the build log is uploaded to Buildkite.
```

Comment:
I can't decide if "Environment variables" or "Values of environment variables" are basically the same thing

Reply:
"Values of environment variables who's names match common patterns" or something?